vars/managerPipeline.groovy

```diff
@@ -158,10 +158,6 @@ def call(Map pipelineParams) {
                                         export SCT_SCYLLA_MGMT_PKG="${params.scylla_mgmt_pkg}"
                                     fi
 
-                                    if [[ ! -z "${params.aws_root_disk_name_monitor}" ]] ; then
```

Comment:
while remove here params from the pipeline, we should remove most of them from here, and leave only the one needed to be called from other jobs
for example, all those can be configure in a per platform yaml, and can be remove from the pipeline params:
```
params.ami_id_monitor
params.ami_monitor_user
params.scylla_repo_m
```

Reply:
the idea of having those in the pipeline, was to git the ability to the dev to put here any manager he wants (as it is running on the monitor node) and be able to set the specific `user` and the `back end scylla` used by the `manager`

Reply:
@fgelcer So should I leave it there?

Reply:
@fgelcer can you point me to where those pipeline params are use in Jenkins.... I doubt they were ever used

Reply:
i don't think we need to leave it